deploy/packages/bbp-packages.yaml

```diff
@@ -23,7 +23,7 @@ packages:
     specs:
       - archngv@0.0.0
       - asciitoh5@1.0
-      - brainbuilder@0.12.1
+      - brainbuilder@0.13.0
```

Comment:
when we do this, is 0.12.1 still in unstable ?

Reply:
I don't know to be honest but `0.12.1` is now a part of `archive/2020-06`
```
module load archive/2020-06
-bash-4.2$ module avail |& grep brainbuilder
brainbuilder/0.11.7                 py-bglibpy/4.2.23
brainbuilder/0.12.0                 py-bluepy/0.14.6
```

Reply:
We will need to check for the snakemake then

Reply:
> when we do this, is 0.12.1 still in unstable ?

Yes. Software does not get deleted… they will just drop out of "scope".